src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1530,15 +1535,47 @@ void ApplicationManagerImpl::OnServiceEndedCallback(
   }
 }
 
+void AddAppIDForRPCService(bool& is_first_rpc_service_accepted,
```

Comment:
@mked-luxoft output parameter was not marked as output, and it is in the begining.  Why nit just return it?

Reply:
@mked-luxoft also function requires description